packages/siwe/lib/client.ts

```diff
@@ -1,10 +1,8 @@
 // TODO: Figure out how to get types from this lib:
-import {
-  isEIP55Address,
-  ParsedMessage,
-  parseIntegerNumber,
-} from '@spruceid/siwe-parser';
-import * as uri from 'valid-url';
+// import { ParsedMessage, parseIntegerNumber } from '@spruceid/siwe-parser';
```

Comment:
This is a monorepo managed with `lerna`. If you run `npm run build` or `npm run test` it will build/test all the packages in the workspace.

Reply:
Got it. It worked. Thanks.

Reply:
Running `npm run build` in the root directory fixed the
> import { ParsedMessage, parseIntegerNumber } from '@spruceid/siwe-parser';

problem. However, I have now added
> export const isUri = (uri: string, doTrace = false) => {}

to the same file, `packages/siwe-parser/lib/abnf.ts`, but 
> import { isUri } from '@spruceid/siwe-parser';

fails even after re-running `npm run build`. I suspect that there is a file somewhere that specifically adds these class and function names to be found in `@spruceid/siwe-parser` but I’ll need some help on that.

Reply:
The entrypoint for the package is `lib/parsers.ts`

Reply:
Got it. Thanks.